### PR TITLE
Add a short guide how to read log-bundles

### PR DESCRIPTION
**Pull request by vrutkovs**
This adds a new `troubleshooting` directory with technical details about debugging components.
First guide is related to log-bundles we use to debug failed installations.

### CODE REVIEW

**Comment by fortinj66** [comment]
Once I’m back from vacation I’ll take a look. 

—John

File: troubleshooting/log-bundle.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+## Data included in log bundle
+
+Manifests
+Systemd logs from bootstrap / masters
+Container inspect / logs
+
+## Where to start reading a log bundle
+
+### Payload and installer version
+First its crucial to ensure the installer and release payload versions match:
```
Comment (LorbusChris) [inline troubleshooting/log-bundle.md:16]:
```suggestion
Firstly it's crucial to ensure the installer and release payload versions match:
```

File: troubleshooting/log-bundle.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  invoker: openshift-internal-ci/release-openshift-okd-installer-e2e-aws-upgrade/1396004378611027968
+  version: unreleased-master-4619-g4b15f0537ad76206c8ba9ba042b404f8c00a295a
+kind: ConfigMap
+metadata:
+  creationTimestamp: null
+  name: openshift-install-manifests
+  namespace: openshift-config
+```
+
+`invoker` format is `unreleased-<branch>-<number of commits>-g<commit hash>`. As you can see,
```
Comment (LorbusChris) [inline troubleshooting/log-bundle.md:39]:
```suggestion
The format of the `invoker` field is `unreleased-<branch>-<number of commits>-g<commit hash>`. As you can see,
```

File: troubleshooting/log-bundle.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+May 22 07:34:46 ip-10-0-6-51 podman[813]: 2021-05-22 07:34:46.807049267 +0000 UTC m=+0.809083790 image pull  
+May 22 07:34:46 ip-10-0-6-51 release-image-download.sh[813]: cfd14e030756425d5f7b09f44926c50c87256f5dd75303f845c27e22790d423e
+May 22 07:34:46 ip-10-0-6-51 systemd[1]: Finished Download the OpenShift Release Image.
+```
+
+It should be inspectable with `oc adm release info <payload pullspec>`.
+
+After release image has been pulled, bootstrap node extracts Machine Config Daemon and updates plain 
+FCOS to OKD content in `release-image-download.service` by pulling `machine-os-content` image from 
+the payload, extracting ostree commit and pivoting bootstrap into it.
```
Comment (LorbusChris) [inline troubleshooting/log-bundle.md:59]:
```suggestion
After a release image has been pulled by `release-image-download.service`, the bootstrap node extracts
the Machine Config Daemon (MCD) from it and updates the plain Fedora CoreOS (FCOS) it has booted
from to a version adjusted for the specific version of OKD. The service deploys it on the bootstrap node by
using `podman` to pull the `machine-os-content` image specified in the payload from the referenced registry, extracting the OSTree commit embedded in the image, and re-booting the OS into the desired version (this
is also called _pivoting_).
```

Comment (vrutkovs) [reply troubleshooting/log-bundle.md:59]: Perhaps we should describe how OKD boot differs from OCP boot - pivoting bootstrap / nodes - and then proceed to troubleshooting instructions?

**Comment by fortinj66** [comment]
Looks good to me

**Pull request merged**